json-rpc/types/src/views.rs

```diff
@@ -26,6 +26,9 @@ use serde::{Deserialize, Serialize};
 use std::{collections::BTreeMap, convert::TryFrom};
 use transaction_builder::get_transaction_name;
 
+pub const JSONRPC_LIBRA_VERSION: &str = "libra-version";
```

Comment:
Should we call it something like 'ledger_version' / 'ledger_timestampused'?
When I first seen libra-version I thought it means version of the node software

Reply:
yeah we can do that, but google json style guide actually advocate for camel case naming, do you think we should do snake_case or camelCase ?

most of the other api name is snake_case, I think we should do it here too

Reply:
Oh, I am ok with either snake or camel, I just meant to call prefix 'ledger', not 'libra', because libra-version sounds more like 'version of libra software'. So I think 'ledger-version', better represents what this field contains, then 'libra-version'

Reply:
yeah I got that, but think we still want libra_ prefix, as an namespace  to avoid potential collision , so I'm proposaling libra_ledger_version  and libra_ledger_timestampusecs instead.

Reply:
That sounds good to me